crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1845,6 +1844,22 @@ impl Room {
             .power_levels())
     }
 
+    /// Gets the suggested role for the user with the provided `user_id`.
+    /// This method checks the `RoomPowerLevels` events instead of loading the
+    /// member list and looking for the member.
```

Comment:
Thanks for writing doc comments <3

Can you add a blank line between the first line (that's supposed to be a short summary) and the rest of the paragraph? This way it displays better in LSP hover hints.

Reply:
Done in https://github.com/matrix-org/matrix-rust-sdk/pull/3170/commits/2030951d1df2577539d117787c883ae9db6e5cbc